en/docs/install-and-setup/prerequisites.md

```diff
@@ -1,14 +1,15 @@
-WSO2 Open Banking UK Toolkit is a purpose-built solution for regulatory compliance that supports PSD2. Instead of 
-building an open banking solution from scratch, you can use WSO2 Open Banking UK Toolkit to meet all open 
-banking requirements with additional benefits beyond compliance. This toolkit runs on top of the following WSO2 products:
+WSO2 Open Banking CDS Toolkit is a purpose-built solution for regulatory compliance that supports Australia Consumer
+Data Right and Consumer Data Standard. Instead of building an open banking solution from scratch, you can use WSO2 Open 
+Banking CDS Toolkit to meet all open banking requirements with additional benefits beyond compliance. This toolkit runs 
+on top of the following WSO2 products:
```

Comment:
Consumer Data `Standards`

Reply:
Hi @silvashenol20 I have fixed this in ce063b3d25eca4f1a31f24e91ede0951a378c105.